### PR TITLE
Render AbsoluteDashing as literal-pixel dashes, not solid lines

### PR DESCRIPTION
**Pull request by ad-si**
## Summary

As part of the scheduled "check a random Wolfram Demonstration against Woxi Studio" task, I downloaded a random Demonstration ("Mamikon's Proof of the Pythagorean Theorem") from the Wolfram Demonstrations Project and validated it against Woxi Studio's notebook parsing/rendering pipeline using the project's headless `dump_notebook`/`dump_manipulate` examples.

The notebook parsed and its `Manipulate` reconstructed successfully, but its interactive graphics use `AbsoluteDashing[{1, 3}]` for the dotted tangent-line construction. The SVG graphics renderer's directive parser (`src/functions/graphics.rs`) had no `"AbsoluteDashing"` case, so it silently fell through as unrecognized and the line rendered **solid** instead of dashed — unlike the already-supported `Dashing`/`AbsoluteThickness` directives.

- Added an `"AbsoluteDashing"` case to `apply_directive`, mirroring `Dashing` but treating every length as literal pixels (the same relationship `AbsoluteThickness` has to `Thickness`), reusing the existing negative-value convention `dash_attr` already understands.
- Added a regression test (`absolute_dashing_produces_dasharray`) verifying `AbsoluteDashing` now produces a `stroke-dasharray` in the exported SVG, alongside the equivalent `Dashing` case and edge cases (`{}`, single-element list).

No source code from the downloaded (copyrighted) Demonstration notebook was copied into the repo — only the general bug it exposed was reproduced with original, minimal test code.

## Test plan
- [x] `cargo test --test interpreter_tests absolute_dashing_produces_dasharray` passes
- [x] Full workspace test suite (`cargo test --workspace`) passes, aside from two pre-existing, unrelated failures (`reciprocal_trig_last_bit::*`) confirmed to also fail on `main` without this change
- [x] `make format`

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_015gBx6NWSWnk829UMBm1YL7

---
_Generated by [Claude Code](https://claude.ai/code/session_015gBx6NWSWnk829UMBm1YL7)_

### CODE REVIEW

File: src/functions/graphics.rs [hunk 1]
```diff
@@ -1495,6 +1495,36 @@ fn apply_directive(expr: &Expr, style: &mut StyleState) -> bool {
...
+              style.dashing = Some(dashes);
+            }
+          }
+          _ => {
+            if let Some(d) = dash_size_to_f64(&args[0])
+              .or_else(|| expr_to_f64(&args[0]).map(|d| -d))
+            {
+              style.dashing = Some(vec![d, d]);
+            }
+          }
```
[the remark this answers is not in the document]
Comment (ad-si) [reply src/functions/graphics.rs:1524]: Good catch, fixed in d7bc5fa — negating the absolute value (`-d.abs()`) instead of the raw parsed number in both branches, plus a regression test for `AbsoluteDashing[{-4, 6}]`.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by ad-si** [comment]
The `test` check is failing on two unrelated tests — `reciprocal_trig_last_bit::cot_takes_the_reciprocal_of_tan` and `::the_other_reciprocals_are_unchanged` (last-bit float mismatches on `Cot[0.3]`/`Coth[0.8]`, e.g. `3.2327281437658275` vs `3.232728143765828`). This isn't caused by this PR: `main`'s own CI at the same base commit (`9293256`) [fails identically](https://github.com/ad-si/Woxi/actions/runs/34710374679/job/103597939063) on the same two tests, and I independently reproduced it locally with this PR's diff stashed out.

No fix exists yet for this pre-existing precision issue, and it's unrelated to the `AbsoluteDashing` change here, so I'm not widening this PR to fix it. Re-running the check once to confirm.

---
_Generated by [Claude Code](https://claude.ai/code)_
